1031-table-design-guidance.md

```diff
@@ -0,0 +1,22 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-22
+updated: 2024-10-22
+---
+
+# Table Component
+
+[Table](https://clarity.design/documentation/table) is a component that use rows and columns to display information in a grid view.
+
+## Guidance
+
+- **Must** use the table component wherever you need to present static data in a tabular format.
+- **Must** alternatively choose [datagrid](https://clarity.design/documentation/datagrid) for advanced features like sorting, filtering, pagination, or perform batch operations on the structured data.
```

Comment:
Design teams **Must** alternatively use the [datagrid](https://clarity.design/documentation/datagrid) for advanced features such as sorting, filtering, pagination, or performing batch operations on structured data. This ensures functionality aligns with user needs and the design system.